dark/index.html

```diff
@@ -194,33 +213,36 @@ <h1>CONTACT ME</h1>
       <h3>CONTACT <span class="text-info">ME</span></h3>
 
       <div class="formContainer bg-dark p-5 rounded-3">
-          <form class="m-auto">
-            <div class="mb-3">
-              <input type="text" class="form-control bg-dark text-white" placeholder="Enter Your Name" id="exampleInputPassword1" />
-            </div>
-            <div class="mb-3">
-              <input type="email" class="form-control bg-dark text-white" placeholder="Enter Your Email" id="email" aria-describedby="emailHelp" />
-            </div>
-            <div class="mb-3">
-              <textarea name="message" id="message" row="20" col="20" class="form-control bg-dark text-white" placeholder="Enter Your Message"></textarea>
-            </div>
-            <button type="submit" class="btn btn-primary d-block m-auto">
-              Submit
-            </button>
-          </form>
-        </div>
-        <center>
-          <p class="contact-text">
-            Contact me via Email, Message or connect me with social media
-            accounts. Also send me any inquiries by filling the form given above.
-          </p>
-          <p class="contact-text">
-            <i class="fa fa-envelope px-1"></i>Email (Personal) : yoshithasbrathnayake@gmail.com
-          </p>
-          <br />
-        </center>
+        <form class="m-auto">
+          <div class="mb-3">
+            <input type="text" class="form-control bg-dark text-white" placeholder="Enter Your Name"
+              id="exampleInputPassword1" />
+          </div>
+          <div class="mb-3">
+            <input type="email" class="form-control bg-dark text-white" placeholder="Enter Your Email" id="email"
+              aria-describedby="emailHelp" />
+          </div>
+          <div class="mb-3">
+            <textarea name="message" id="message" row="20" col="20" class="form-control bg-dark text-white"
+              placeholder="Enter Your Message"></textarea>
+          </div>
+          <button type="submit" class="btn btn-primary d-block m-auto">
+            Submit
+          </button>
+        </form>
       </div>
-    </section>

```

Comment:
Why did you remove this section ending tag? @priyankarpal

Reply:
checking...